avb/libavb_user/uefi_avb_ops.c

```diff
@@ -273,11 +269,9 @@ static AvbIOResult write_rollback_index(__attribute__((unused)) AvbOps* ops,
   if (is_live_boot())
     ret = EFI_SUCCESS;
   else {
-#if defined(SECURE_STORAGE_EFIVAR)
 #ifdef USE_TPM
-    if (!is_platform_secure_boot_enabled() ||
-        (ret = write_rollback_index_tpm2(rollback_index_slot, rollback_index)) == EFI_NOT_FOUND)
-#endif // USE_TPM
+      ret = write_rollback_index_tpm2(rollback_index_slot, rollback_index);
+#else
       ret = write_efi_rollback_index(rollback_index_slot, rollback_index);
```

Comment:
Since we removed if condition check, need to move 2 indents forward. (remove 2 spaces in front of this line)